email_validator.gemspec

```diff
@@ -12,15 +12,12 @@ Gem::Specification.new do |s|
     README.md
     CHANGELOG.md
   ]
-  s.files = `git ls-files -- lib/*`.split("\n")
```

Comment:
This backquote section occurs build error on JRuby.

Reply:
Is that a bug in JRuby? What does it cause a build error?